__tests__/integration.test.js

```diff
@@ -259,4 +259,50 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(msg).toBe("username random_name not found");
       });
   });
-});
\ No newline at end of file
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: edits the article with the given id", () => {
+    const newVotes = { inc_votes: -100 };
+
+    return request(app)
+      .patch("/api/articles/7")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {         
+        expect(article.article_id).toBe(7)     
+        expect(article).toEqual(
+          expect.objectContaining({
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Instead of asserting the article_id separately before you can have this here instead of the expect.any(...).